Float64 Support (#387)
hms-dbmi/viv: CHANGELOG.md

### Added

> - Support OME `double` (and in general, double-precision floating point) datatype

Reviewer: ```suggestion
 - Support `Float64` (OME `double`) datatype by casting array data to `Float32`. 
```
